firedrake/interpolation.py

```diff
@@ -460,6 +460,9 @@ def composed_map(map1, map2):
     Requires that `map1.toset == map2.iterset`.
     Only currently implemented for `map1.arity == 1`
     """
+    if map2 is None:
+        # Real function space case
+        return None
```

Comment:
Great so this should fix #2095 ? Can you add a test to `tests/regression/test_interpolate.py`?

Reply:
Perhaps the tests which you've re-enables in `tests/vertexonly` are enough

Reply:
I think so, that's how I noticed it.